doc/source/user_guide/cookbook.rst

```diff
@@ -1099,7 +1099,7 @@ HDFStore
 
 The :ref:`HDFStores <io.hdf5>` docs
 
-`Simple Queries with a Timestamp Index
+`Simple queries with a timestamp index
```

Comment:
Timestamp and Index are class names, can you leave the capitalization of those please